scripts/py_matter_idl/matter/idl/README.md

```diff
@@ -26,9 +26,13 @@ cluster AccessControl = 31 {
   // Enums and structs can be defined globally or be cluster specific.
   // IDL generation rules will take into account scoping (i.e. pick local defined
   // name first, things defined in one cluster are not visible in another).
+  //
+  // Constants allow a `spec_name` extra attribute to clarify the name source of
+  // the constant in cases where name generation logic depends on acronym
+  // expansion or not (e.g. special characters, spaces, full uppercase/digit logic)
   enum AuthMode : ENUM8 {
     kPase = 1;
-    kCase = 2;
+    kCase = 2 [spec_name = "CASE"];
```

Comment:
Isn't the IDL actually `kCASE` here?

Reply:
the idl generates `k<NAME>` inside the constants, however for the spec name I kept the exact label without the k prefix as that seemed potentially clearer.

The k was in case something is named "2.4GHz" so that things do not start with digits and are guaranteed to start with a letter.

Reply:
My point is that this line should be `kCASE = 2 [spec_name = "CASE"]` if this is meant to look like the actual AuthModeEnum....  But maybe this is just an example which is not related to the spec, in which case we should not use spec names in it.

Reply:
But yes, this is a 100% made up example. I will re-write it for a real one.